<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetSYNC_dial</class>
 <widget class="QDialog" name="NetSYNC_dial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Команда: &quot;NetWorkSYNC&quot;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Идентификатор ПАК УВС :     </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_DCE_ID">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Идентификатор получателя :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_DCE_ID_2">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Иммитовставка :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_imit">
       <property name="displayIntegerBase">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pb_default">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_applay">
       <property name="text">
        <string>Применить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_cancel">
       <property name="text">
        <string>Отменить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1" rowspan="2" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Синхропакет :</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QSpinBox" name="sb_pack1"/>
        </item>
        <item>
         <widget class="QSpinBox" name="sb_pack2"/>
        </item>
        <item>
         <widget class="QSpinBox" name="sb_pack3"/>
        </item>
        <item>
         <widget class="QSpinBox" name="sb_pack4"/>
        </item>
        <item>
         <widget class="QSpinBox" name="sb_pack5"/>
        </item>
        <item>
         <widget class="QSpinBox" name="sb_pack6"/>
        </item>
        <item>
         <widget class="QSpinBox" name="sb_pack7"/>
        </item>
        <item>
         <widget class="QSpinBox" name="sb_pack8"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_13">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Команда : &quot;NetWorkSYNC&quot;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
